### DISCUSSION TITLE
Welcome to aoc2020 Discussions!

### DISCUSSION THREAD

**Discussion by root42** [root42/aoc2020#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StefanLarsson** [comment]
Ok, time to try out the discussions feature!

Wondering what the big "#2" up there is about. This seems to be the first discussion in this repo. Is it the 2nd entry in this discussion? But that seems weird, since the UI shows that I am commenting on this particular entry, so I expect the discussions to be threaded, and an order number is not so useful. Never mind!

I am Stefan, and my role in this community is to steal ideas from Arnes solutions for the Advent of Code tasks, since I am also trying to use AoC to learn more about Clojure.
